06_deprecations/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Deprecations
+nav_order: 7
+has_children: true
+last_modified_date: 13-12-2021 13:00
+permalink: /deprecations/
+---
+
+# Deprecations
```

Comment:
Could you add a definition of deprecation or an explanation of what it means?

Reply:
Done